src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -228,6 +228,18 @@ ErrorMessageOr<std::unique_ptr<PdbFile>> PdbFileLlvm::CreatePdbFile(
     return ErrorMessage(absl::StrFormat("Unable to load PDB file %s with error: %s",
                                         file_path.string(), llvm::toString(std::move(error))));
   }
+  auto* native_session = dynamic_cast<llvm::pdb::NativeSession*>(session.get());
+  ORBIT_CHECK(native_session != nullptr);
+  llvm::pdb::PDBFile& pdb_file = native_session->getPDBFile();
+  const std::string dbi_error_str = absl::StrFormat(
+      "Unable to load PDB file %s with error: PDB has no Dbi Stream", file_path.string());
```

Comment:
I would drop the "with error" from this message and the one above, it's implied that what follows is a more detailed error description.

Reply:
Done.